step_configure_hw.go

```diff
@@ -36,7 +40,16 @@ func (s *StepConfigureHW) Run(state multistep.StateBag) multistep.StepAction {
 		confSpec.MemoryMB = int64(ram)
 	}
 
-	state.Put("confSpec", confSpec)
+	task, err := vm.Reconfigure(ctx, confSpec)
```

Comment:
Can we make this call optional? If no customization options are specified, then don't reconfigure VM, and don't print a log message.